scopes/ui-foundation/uis/top-bar/top-bar.module.scss

```diff
@@ -9,5 +9,5 @@ $topbarHeight: 56px;
   background: #ededed;
   box-sizing: border-box;
   font-size: var(--bit-p-xs, 14px);
-  z-index: $nav-z-index;
+  z-index: $nav-z-index + 1;
```

Comment:
top bar needs to be with higher zindex in order for the menu to go over collapser button and splitter

Reply:
cool, include this comment